octavia_proxy/tests/unit/api/drivers/elbv3/test_driver.py

```diff
@@ -217,6 +259,11 @@ def test_flavors_no_qp(self):
         self.driver.flavors(self.sess, 'p1')
         self.sess.vlb.flavors.assert_called_with()
 
+    def test_flavor_get(self):
+        self.driver.flavor_get(self.sess, 'test', self.flavor)
+        self.sess.vlb.find_flavor.assert_called_with(
+            name_or_id=self.flavor, ignore_missing=True)
+
 
```

Comment:
too much of new logic and too few tests. Would you please add following:
- list (drivers gives us returns multiple entries and we only return L7 ones)
- get (knowing concrete ID)
- find (passing `s1.medium`)
- find (passing `L7_flavor.elb.s1.medium`) to ensure we return nothing